### ISSUE TITLE
Strange results for joins

### ISSUE THREAD

**Issue by zaiste** [khiav223577/pluck_all#29]
```rb
Post.includes(:user).where(users: { regular: true }).pluck_all(:id, :title)
```

Such query returns some strange results.

**Comment by khiav223577** [comment]
Thanks for you reporting. I will take it a look latter.

As a quick workaround, you can use `left_joins` instead of `includes`.
```rb
Post.left_joins(:user).where(users: { regular: true }).pluck_all(:id, :title)
```

**Comment by khiav223577** [comment]
I just released v2.0.3. I believe it resolves the issue :)
